nibiru/chain.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nibiru",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://nibiru.fi/",
+  "pretty_name": "Nibiru",
+  "chain_id": "cataclysm-1",
+  "bech32_prefix": "nibi",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Pending PR: https://github.com/satoshilabs/slips/pull/1647